helper_scripts/setup_helpers.py

```diff
@@ -23,10 +23,11 @@ def create_input(base_fp: str, engine_props: dict):
     with open(save_path, 'r', encoding='utf-8') as file_object:
         engine_props['mod_per_bw'] = json.load(file_object)
 
-    network_dict = create_network(base_fp=base_fp, const_weight=engine_props['const_link_weight'],
-                                  net_name=engine_props['network'])
+    network_dict, core_nodes = create_network(base_fp=base_fp, const_weight=engine_props['const_link_weight'],
```

Comment:
We might be able to include core nodes in network dict if you want, this would be more scalable because we could add many variables to network_dict and not continue to return new variables